### PR TITLE
Bye 👋

### PR DESCRIPTION
**Pull request by gtsiap**
Delete any references to me from playbook.

### CODE REVIEW

**Review by konrad-em — APPROVED** [review]
😢

File: scripts/squads.yml [hunk 1]
```diff
@@ -106,7 +103,6 @@ squads:
   - name: Experience
     desc: Engage users with our key features in a dynamic, personalised manner.
     members:
-      - *Giorgos
```
Comment (Naiko) [inline scripts/squads.yml:109]: Should we delete that squad altogether?

Comment (sdaniel55) [reply scripts/squads.yml:109]: yes, the script will fail otherwise

**Review by danspndl — APPROVED** [review]
😢

**Pull request merged**